src/EntityGraphQL/Schema/ISchemaProvider.cs

```diff
@@ -27,7 +27,7 @@ public interface ISchemaProvider
         ISchemaType AddInterface(Type type, string name, string description);
         SchemaType<TBaseType> AddInputType<TBaseType>(string name, string? description);
         ISchemaType AddInputType(Type type, string name, string? description);
-        void AddMutationsFrom<TType>(TType mutationClassInstance, bool autoAddInputTypes = false) where TType : notnull;
+        void AddMutationsFrom<TType>(bool autoAddInputTypes = false);
```

Comment:
I try (my best) to avoid breaking changes to the API - trying to follow semver where x.y.z is x - breaking changes, y new features, z fixes. 

Given the change, we could also leave the old methods there and mark them as deprecated. The methods can just do nothing with the object passed in. This wold be a 2.2 release.

Then we can later do a 3 that cleans them up and potentially wraps up more breaking changes.

Thoughts?

Reply:
Would it work if the mutationClassInstance argument remained but defaults to null?